vllm/worker/pooling_model_runner.py

```diff
@@ -110,7 +114,8 @@ def execute_model(
                 intermediate_tensors=intermediate_tensors,
                 **MultiModalKwargs.as_kwargs(multi_modal_kwargs,
                                              device=self.device),
-                **cross_enc_kwargs)
+                **cross_enc_kwargs,
+                **seqlen_agnostic_kwargs)
```

Comment:
Are these changes intentional?

Reply:
yes, we want to add thoses kwargs as well

Reply:
cc @tlrmchlsmth

Reply:
Haven't looked at the rest of the PR but this change in particular looks good to me, assuming `finished_requests_ids` and `request_ids_to_seq_ids` are getting plumbed down properly